upt_macports/templates/perl.Portfile

```diff
@@ -0,0 +1,20 @@
+{% extends 'base.Portfile' %}
+
+{% block portgroup %}
+PortGroup           perl5 1.0
+{% endblock %}
+{% block nameversion %}
+perl5.branches      5.26 5.28
+perl5.setup         {{ pkg._pkgname() }} {{ pkg.upt_pkg.version }}
+revision            0
+{% endblock %}
+
+{% block versions %}
+if {${perl5.major} != ""} {
+    depends_lib-append \
+                    {% for i in pkg.upt_pkg.requirements.run %}
+                    port:p${perl5.major}-{{i.name|replace('::','-')|lower}} {% if loop.index != pkg.upt_pkg.requirements.run|length %}\{% endif %}
```

Comment:
Two things that I don't want to block merging of this PR, but it would be nice to address them later:
* I would use a function to generate names of perl packages (we might need that also for recursive checking if a package exists). That function should not be part of the template.
* Probably the same nitpicking about dependencies as for python (test dependencies missing, sorting, ...). Making me think that this might be one of the things that could potentially be done in base and be repeated for different types of packages (but then it requires more thinking to be done in a way that's general enough).